src/actions/make/_make.lua

```diff
@@ -144,8 +144,17 @@
 		_p('')
 
 		if kind == "workspace" then
-			_p('.NOTPARALLEL:')
-			_p('')
+			local haspch = false
+			for _, prj in ipairs(target.projects) do
+				if prj.pchheader then
```

Comment:
Thanks. Updated the PR to account for pchheaders at the config level. 

Rebased from master.